app/views/searches/index.html.erb

```diff
@@ -10,49 +10,180 @@
 </div>
 
 <script type="text/javascript">
-  $(function(){
-    $('.js-venue-list').imagepicker({
-      show_label: true,
-      changed: function () {
-          $('#filter-form').submit()
-      }
+var refreshURL = '';
```

Comment:
We need to extract this JS code into a saner structure.  There's getting to be way too much of it here.